{"name": "Проверка: если попытаться создать курьера с таким же логином, как у уже существующего, вернётся ошибка 409", "status": "passed", "start": 1710504301588, "stop": 1710504302646, "uuid": "c8f92d51-c9d9-47a8-99e8-21bf7cd47462", "historyId": "e92d7bd1cea5620a7bd97122004273db", "testCaseId": "e92d7bd1cea5620a7bd97122004273db", "fullName": "tests.tests_courier.test_create_courier.TestCreateCourier#test_create_two_same_login_couriers_causes_409_error", "labels": [{"name": "parentSuite", "value": "tests.tests_courier"}, {"name": "suite", "value": "test_create_courier"}, {"name": "subSuite", "value": "TestCreateCourier"}, {"name": "host", "value": "Olyas-MacBook-Air.local"}, {"name": "thread", "value": "78521-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.tests_courier.test_create_courier"}]}